plat/rockchip/common/bl31_plat_setup.c

```diff
@@ -69,8 +71,20 @@ void params_early_setup(void *plat_param_from_bl2)
 void bl31_early_platform_setup(bl31_params_t *from_bl2,
 			       void *plat_params_from_bl2)
 {
-	console_init(PLAT_RK_UART_BASE, PLAT_RK_UART_CLOCK,
-		     PLAT_RK_UART_BAUDRATE);
+	static console_16550_t console;
+
+	params_early_setup(plat_params_from_bl2);
+
+#if COREBOOT
+	if (coreboot_serial.type)
+		console_16550_register(coreboot_serial.baseaddr,
+				       coreboot_serial.input_hertz,
+				       coreboot_serial.baud,
+				       &console);
+#else
+	console_16550_register(PLAT_RK_UART_BASE, PLAT_RK_UART_CLOCK,
```

Comment:
Static check failure:
Return value of non-void function console_16550_register() is not used.

Reply:
Is this a blocking error? The old code here was also calling console_init() (which returned an int) without checking the result. I'm essentially not changing anything about that, just adjusting this to match the slightly changed API (but the return value still means the same).

Also, I don't think there's anything really useful you can do about an error here.

Reply:
@jwerner-chromium 
Ok - we can ignore this issue.